app/components/task-view.vue

```diff
@@ -151,186 +154,256 @@ useMagicKeys({
     }
   },
 });
+
+const onFileDropped = async (files: File[]) => {
+  if (!task.value) {
+    return;
+  }
+
+  for (const file of files) {
+    await addAttachment({
+      taskId: task.value.id,
+      workspaceId: task.value.workspaceId,
+      columnId: task.value.statusColumnId,
+      file,
+    })
+      .then(() => {
+        toast.success(`File ${file.name} uploaded successfully`);
+      })
+      .catch((error: FetchError) => {
+        toast.error(error.message);
+      });
+  }
```

Comment:
Make these parallel (Promise.all/settletd).

Make toast messages not disappear automatically.